### PR TITLE
replacement logo and footer

### LINKED ISSUE #13: Modify snapshots to clearly declare that they were not created by wikipedia
If we publish unmodified snapshots of wikipedia it may confuse visitors, leading them to believe that the snapshots were published by Wikipedia or by volunteers who contribute to WIkipedia in Turkey. This could present a real risk for those volunteers, who have nothing to do with our snapshots, and may harm wikipedia, who did not ask us to do this.

I propose modifying the snapshots to: 

1. Remove references to the Wikimedia user group in Turkey from the bottom of the page

2. Adjust the top of the page to distinguish it from Wikipedia (by removing the puzzle globe logo and adding a prominent explanation that this is an independent project, not affiliated with Wikipedia)

This will force us to scrap all of the snapshots we've built and start over with modified code, which will delay the release of the snapshots by at least a few days.

### PR DESCRIPTION
**Pull request by flyingzumwalt**
Based on suggestions from https://github.com/ipfs/distributed-wikipedia/issues/13

To do:
- [x] fill in the second footer with social media links
- [x] make a `body.js` that inserts our footer where wikipedia's original body.js inserts the social media links. (#17)
- [x] add a new Distributed Wikipedia Mirror logo that doesn't have the wikipedia puzzle-piece globe inside the ipfs cube (#16 - @jbenet)

closes #13

### CODE REVIEW

**Comment by Kubuxu** [comment]
We can't have hashes of the wiki itself in the wiki. Chicken and egg problem. We can use ipns names.

**Comment by VictorBjelkholm** [comment]
Pushed an update, looks like this now:

image

There is a huge amount texts in the first part of the footer though, and if possible we should reduce the amount of content

File: templates/footer.html [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
+	This page was generated and distributed by the <a class="external text" href="https://github.com/ipfs/distributed-wikipedia">Distributed Wikipedia project</a>. It's based on the article issued from <a class="external text" href="https://tr.wikipedia.org/wiki/{{ARTICLE_NAME}}?oldid={{ARTICLE_ID}}">Vikipedi</a> - version of the {{ARTICLE_DATE}}. The text is available under the <a class="external text" href="http://creativecommons.org/licenses/by-sa/3.0/">Creative Commons Attribution/Share Alike</a> but additional terms may apply for the media files. The Distributed Wikipedia Project is not affiliated to Wikipedia in any way.
   </p>
   <p>
     From Snapshot: /ipfs/QmQP99yW82xNKPxXLroxj1rMYMGF6Grwjj2o4svsdmGh7S<br/>
-    Created on: 2017-05-05<br/>
+		Created on: {{ARTICLE_DATE}}<br/>
     IPFS Link: dweb://ipfs/QmQP99yW82xNKPxXLroxj1rMYMGF6Grwjj2o4svsdmGh7S/out/A/Anasayfa.html<br/>
     HTTP Link: <a class="external text" href="https://ipfs.io/ipfs/QmQP99yW82xNKPxXLroxj1rMYMGF6Grwjj2o4svsdmGh7S/out/A/Anasayfa.html">https://ipfs.io/ipfs/QmQP99yW82xNKPxXLroxj1rMYMGF6Grwjj2o4svsdmGh7S/out/A/Anasayfa.html</a><br/>
```
Comment (Kubuxu) [inline templates/footer.html]: This should probably just be `/ipfs/Qm...`

File: templates/footer.html [hunk 2]
```diff
@@ -1,21 +1,23 @@
...
+	This page was generated and distributed by the <a class="external text" href="https://github.com/ipfs/distributed-wikipedia">Distributed Wikipedia project</a>. It's based on the article issued from <a class="external text" href="https://tr.wikipedia.org/wiki/{{ARTICLE_NAME}}?oldid={{ARTICLE_ID}}">Vikipedi</a> - version of the {{ARTICLE_DATE}}. The text is available under the <a class="external text" href="http://creativecommons.org/licenses/by-sa/3.0/">Creative Commons Attribution/Share Alike</a> but additional terms may apply for the media files. The Distributed Wikipedia Project is not affiliated to Wikipedia in any way.
   </p>
   <p>
     From Snapshot: /ipfs/QmQP99yW82xNKPxXLroxj1rMYMGF6Grwjj2o4svsdmGh7S<br/>
```
Comment (Kubuxu) [inline templates/footer.html]: I still don't know where we should get this hash from. We could in theory get it with some JS magic but we can't just straight embed it as it has recursive dependency onto itself.

You can't have something with its hash inside.

Comment (VictorBjelkholm) [inline templates/footer.html]: Yeah, what we can have is the IPNS address. However, I don't think we can even do any JS magic to get the hash, without making assumptions about where the hash is in location

**Comment by flyingzumwalt** [comment]
As observed [here](https://github.com/ipfs/distributed-wikipedia-mirror/issues/13#issuecomment-299579934) I think we can get away with
* replacing the content of `/I/s/Wikipedia-logo-v2-200px-transparent.png` with the Distributed Wikimedia Mirror logo from https://github.com/ipfs/distributed-wikipedia-mirror/blob/feat/footer/assets/distributed-wikipedia.png (leave the original filename of Wikipedia-logo-v2-200px-transparent.png) -- _might require resizing the image_
* replacing the content of `/out/-/j/body.js` with javascript that generates the footer html we've created in https://github.com/ipfs/distributed-wikipedia-mirror/blob/feat/footer/templates/footer.html

With this approach, we can just change those 2 files, re-add the snapshot with those 2 files changed, and publish the new root hash.

**Comment by whyrusleeping** [comment]
@flyingzumwalt thats great news! That means the hashes of most of the content won't have to change

**Comment by flyingzumwalt** [comment]
exactly!

**Comment by flyingzumwalt** [comment]
@VictorBjelkholm could you take a stab at making a version of body.js that writes the footer.html into the document?

File: templates/footer.html [hunk 3]
```diff
@@ -8,7 +8,7 @@
...
+    This page was generated and distributed by the <a class="external text" href="https://github.com/ipfs/distributed-wikipedia-mirror">Distributed Wikipedia Mirror project</a>, from the article <a class="external text" href="https://{{LANGUAGE_CODE}}.wikipedia.org/wiki/{{ARTICLE_NAME}}?oldid={{ARTICLE_ID}}">{{ARTICLE_NAME}}</a>, as of {{ARTICLE_DATE}}. For the most recent version of the article, see <>. The text is available under a <a class="external text" href="http://creativecommons.org/licenses/by-sa/3.0/">Creative Commons Attribution/Share Alike</a> license, by Wikipedia authors. Media files are available under similar licenses; see the original file for details. The Distributed Wikipedia Mirror is a global effort, independent from Wikipedia.
```
Comment (Kubuxu) [inline templates/footer.html]: ` For the most recent version of the article, see <>.` seems broken a bit.

Comment (flyingzumwalt) [inline templates/footer.html]: yep. took that out of the most recent commit.

File: templates/footer.html (hunk 2 above)
Comment (flyingzumwalt) [inline templates/footer.html]: I updated this section to use the IPNS hash, which can be generated into the template before building a snapshot. I assume that the IPNS hash will vary based on the language we're building a snapshot of.

File: templates/footer.html (hunk 1 above)
Comment (flyingzumwalt) [inline templates/footer.html]: The idea here is to have working http links that are valid if you click _and_ if you copy and paste.

**Comment by jbenet** [comment]
See https://github.com/ipfs/distributed-wikipedia-mirror/issues/16 for updated logo

**Comment by jbenet** [comment]
Could we also:
- [ ] make the `dweb://` links be clickable? fine if they dont work yet.
- [ ] use the nice Wikipedia font for the footer?  (http://www.dafont.com/linux-libertine.font) (we dont have to do this but would be nice)
- [ ] use social icons that fit together? (i chose some square ones free to use for this purpose)
- [ ] remove redundant line ("it was derived from vikipedi - version of the ... apply for the media files" -- show it once)

**Comment by VictorBjelkholm** [comment]
@flyingzumwalt yes, will be looking into it today

**Comment by flyingzumwalt** [comment]
The positioning of the Wikipedia-logo-v2-200px-transparent.png background image in the banner is hard-coded into the `<tr>` style attribute, so I made a version of the logo that corrects for the offset. It's ugly but it works. The offset logo file is in `assets/replace-Wikipedia-logo-v2-200px-transparent.png` See #18 for my thoughts on how to add it to the snapshots.

**Comment by flyingzumwalt** [comment]
I'm going to merge this as-is. It would be best if we can clean up the logos, but I want to track that with separate issues rather than letting this PR sprawl.

**Pull request merged**